### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sabian-heatwave**
When I first started working on [grAyG](https://github.com/thephoenixprojectsoftware/grayg), I had trouble figuring out how to build it, so I made this README for beginners with CMake.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,12 +8,15 @@ OpenAG is an open-source client of the Half-Life promod Adrenaline Gamer, comple
...
-1. Get Visual Studio 2017 and CMake.
-2. `git submodule update --init`
-3. `mkdir build`
-4. `cd build`
-5. `cmake ..`
-6. `cmake --build . --config Release`
+1. Install Visual Studio 2017 from https://my.visualstudio.com/Downloads?q=Visual%20Studio%202017 (make sure you select Visual Studio 2017 Community version 15.9). In the Visual Studio Installer, select Desktop Development for C++ and not anything else.
+2. Install Git from https://git-scm.com/. Make sure you add it to PATH.
+3. Install CMake from https://cmake.org. Make sure you add it to PATH.
+4. Go to where you saved this cloned repo, right click ans select 'Git Bash Here.' In Git, type: `git submodule update --init`
```
Comment (YaLTeR) [inline README.md]: If you're explaining how to download and install VS, Git and CMake then you shouldn't skip the "cloning the repository" step.

File: README.md [hunk 2]
```diff
@@ -8,12 +8,15 @@ OpenAG is an open-source client of the Half-Life promod Adrenaline Gamer, comple
...
+5. `mkdir build`
+6. `cd build`
+7. `cmake .. -G "Visual Studio 15 2017"`
+8. `cmake --build . --config Release`
+9. The CMake program will generate the OpenAG.sln file that you can use with Visual Studio 2017. Make sure you DO NOT build using VS2017; save the files you have changed and run the `cmake --build . --config Release` command. The built DLL will be in the Release folder.
```
Comment (YaLTeR) [inline README.md]: Huh, what's wrong with building using VS?

Comment (sabian-heatwave) [reply README.md]: Damn, only just now I found out you could build using VS too, I'll make the changes right away

Comment (sabian-heatwave) [reply README.md]: I've made the changes requested, should be fine now :)

**Comment by YaLTeR** [comment]
So I went into VS 2017 to verify the procedure, and I think I have found a simpler way without having to install Git or CMake, and without using the command-line at all:
1. Open Visual Studio 2017.
1. Go to File→Open→Open from Source Control.
1. Click Clone:
    image
     Enter `https://github.com/YaLTeR/OpenAG.git`, press the Clone button. Wait for the process to finish.
1. Click on the drop-down at the top and select "Manage Configurations...":
    image
1. Click "x86-Debug" and press the Select button. The drop-down should now say "x86-Debug"; if it doesn't, go to File→Close Folder and then open it again.
1. You can build the project using CMake→Build All. To find the built client.dll, go to CMake→Cache (x86-Debug Only)→Open Cache Folder→OpenAG.

The build, done in this fashion, currently fails with
```
C:\Users\YaLTeR\Desktop\OpenAG\cl_dll\update_checker.cpp(125): error C2065: 'ull': undeclared identifier
```
but that's something I can try to fix.

Could you try if this works for you? Do you think this would be a better way for beginners?

**Comment by YaLTeR** [comment]
Actually, I think I can commit the configuration settings file which would make 2 of these steps unnecessary.

**Comment by sabian-heatwave** [comment]
> So I went into VS 2017 and 2019 to verify the procedure, and I think I have found a simpler way without having to install Git or CMake, and without using the command-line at all:
> 
> ### Visual Studio 2019
> 1. Open Visual Studio.
> 2. On the starting screen, click "Clone or check out code".
> 3. Enter `https://github.com/YaLTeR/OpenAG.git`, press the Clone button. Wait for the process to finish.
> 4. You can build the project using Build→Build All. To find the built client.dll, go to Project→CMake Cache (x86-Debug Only)→Open in Explorer.
> 
> ### Visual Studio 2017
> 1. Open Visual Studio.
> 2. Go to File→Open→Open from Source Control.
> 3. Click Clone:
>    image
>    Enter `https://github.com/YaLTeR/OpenAG.git`, press the Clone button. Wait for the process to finish.
> 4. You can build the project using CMake→Build All. To find the built client.dll, go to CMake→Cache (x86-Debug Only)→Open Cache Folder→OpenAG.
> 
> Could you try if this works for you? Do you think this would be a better way for beginners?

I will try this now; I do think it would be better for beginners

**Comment by sabian-heatwave** [comment]
@YaLTeR I used VS2017 and 2019 with no issues without Git and CMake and I have updated the README.md.

**Comment by YaLTeR** [comment]
Thanks! I'll tweak the readme a little more to my liking.

**Pull request merged**